src/pages/History/History.jsx

```diff
@@ -1,5 +1,43 @@
 import React from "react";
+import { Navbar, Sidebar, VideoCard } from "components";
+import styles from "./history.module.css";
+import { useVideosOperations } from "context";
 
 export function History() {
-  return <div>History</div>;
+  const {
+    videosOperations: { historyVideos },
+    manageDeleteHistoryAll,
+  } = useVideosOperations();
+  const btnDeleteHandler = () => {
+    manageDeleteHistoryAll();
+  };
+
+  return (
+    <div className={styles.container}>
+      <section className={styles.navbar}>
+        <Navbar />
+      </section>
+      <section className={styles.sidebar}>
+        <Sidebar />
+      </section>
+
+      <main className={styles.main}>
+        <div className={styles.topBar}>
+          {historyVideos.length > 0 ? (
+            <button
+              className='btn btn-danger btn-secondary'
+              onClick={btnDeleteHandler}
```

Comment:
Since this is clear all history function , instead of naming btnDeleteHandler , DeleteAllHistory something , to make it clear - Suggestion

Reply:
Sure will make the changes.